docs/compression_algorithms/Quantization.md

```diff
@@ -310,116 +310,114 @@ Following is an example of wrapping ImageNet validation loop as a callback. Top5
-            "tolerance": 1e-4, //  Minimum relative tolerance for stopping the Hutchinson algorithm. It's calculated between mean average trace from previous iteration and current one. 1e-4 by default
-            "compression_ratio": 1.5, // The desired ratio between bits complexity of fully INT8 model and mixed-precision lower-bit one.
-            "bitwidth_per_scope": [ // Manual settings for the quantizer bitwidths. Scopes are used to identify the weight quantizers. The same number of bits is assigned to adjacent activation quantizers. By default bitwidth is taken from global quantization parameters from `weights` and `activations` sections above
-                [
-                    4,
-                    "InsertionType.NNCF_MODULE_PRE_OP MobileNetV2/Sequential[features]/InvertedResidual[16]/Sequential[conv]/NNCFConv2d[2]"
-                ], // A tuple of a bitwidth and a scope
-                [
-                    4,
-                    "TargetType.OPERATOR_POST_HOOK MobileNetV2/Sequential[features]/ConvBNReLU[0]/ReLU6[2]/hardtanh_0",
-                ]
-            ]
-        }
-        "batchnorm_adaptation": {
-            "num_bn_adaptation_samples": 2048, // Number of samples from the training dataset to pass through the model at initialization in order to update batchnorm statistics of the original model. The actual number of samples will be a closest multiple of the batch size.
-        }
+    "input_info": { "sample_size": [1, 3, 224, 224] }, // the input shape of your model may vary
+    "compression": {
+       "algorithm": "quantization",
     }
-    "weights": { // Constraints to be applied to model weights quantization only.
-        "mode": "symmetric", // Mode of quantization
-        "bits": 8, // Bitwidth to quantize to. It is intended to manually specify bitwidth for all weights. Can be overridden by the `bits` parameter from the `precision` initializer section. An error happens if it doesn't match a bitwidth constraints for module weight specified in the hardware configuration.
-        "signed": true, // Whether to use signed or unsigned input/output values for quantization. If specified as unsigned and the input values during initialization have differing signs, will reset to performing signed quantization instead.
-        "per_channel": false, // Whether to quantize inputs per channel (i.e. per 0-th dimension for weight quantization,and per 1-st dimension for activation quantization)
-
-        // A list of model control flow graph node scopes to be ignored for this operation - functions as a 'denylist'. Optional.
-        "ignored_scopes": []
+}
+```
 
-        // A list of model control flow graph node scopes to be considered for this operation - functions as a 'allowlist'. Optional.
-        // "target_scopes": []
+- Quantize a model to 8-bit precision targeted for Intel CPUs, with additional constraints of symmetric weight quantization and asymmetric activation quantization:
+```json5
+{
```

Comment:
There is quite a lot of information in this document. Can we have separate the examples of configs and place them after the corresponding sections: uniform quantization (8-bits) + its samples, HAWQ + its samples, AutoQ + its samples, etc.

Reply:
It would I guess. My main message here is that it does not make sense to read/scroll through the whole section if I need to try quantization for my model, especially considering that the two methods at the bottom are experimental.

Reply:
Made sure that the sections contain at least one link to the full configs in the `examples` folder.